operator/v1/types_network.go

```diff
@@ -349,6 +353,35 @@ type HybridOverlayConfig struct {
 type IPsecConfig struct {
 }
 
+type PolicyAuditConfig struct {
+	// RateLimit is the approximate maximum number of messages to generate per-second per-node. If
+	// unset the default of 20 msg/sec is used.
+	// +optional
+	RateLimit *uint32 `json:"rateLimit,omitempty"`
+
+	// MaxFilesSize is the max size an ACL_audit log file is allowed to reach before rotation occurs
+	// Default is 50MB
+	// +optional
+	MaxFileSize *uint32 `json:"maxFileSize,omitempty"`
```

Comment:
which unit? So default is 50000000 ? Are you sure about uint32? 

Is zero a valid file size?

Reply:
I think I will Try to change the unit here to MB, 0 should not be valid